lib/screens/home_screen.dart

```diff
@@ -29,141 +29,137 @@ class HomeScreen extends StatefulWidget {
+class HomeScreenContent extends StatelessWidget {
+  const HomeScreenContent({
+    Key? key,
+  }) : super(key: key);
 
-      case 1:
-        easelProvider.willLoadFirstTime = true;
-        break;
-      case 2:
-        easelProvider.willLoadFirstTime = false;
-        break;
-    }
+  @override
+  Widget build(BuildContext context) {
+    final homeViewModel = context.watch<HomeViewModel>();
+    return Column(
+      children: [
+        const VerticalSpace(20),
+        MyStepsIndicator(currentPage: homeViewModel.currentPage, currentStep: homeViewModel.currentStep),
+        const VerticalSpace(5),
+        StepLabels(currentPage: homeViewModel.currentPage, currentStep: homeViewModel.currentStep),
+        const VerticalSpace(10),
+        Stack(
+          alignment: Alignment.center,
+          children: [
+            Align(
+                alignment: Alignment.centerLeft,
+                child: ValueListenableBuilder(
+                  valueListenable: homeViewModel.currentPage,
+                  builder: (_, int currentPage, __) => Padding(
+                      padding: EdgeInsets.only(left: 10.sp),
+                      child: IconButton(
+                        onPressed: () {
+                          ScaffoldMessenger.of(context).hideCurrentSnackBar();
+                          homeViewModel.pageController.previousPage(duration: const Duration(milliseconds: 300), curve: Curves.easeIn);
+                          if (homeViewModel.currentPage.value == 0) {
+                            GetIt.I.get<CreatorHubViewModel>().getDraftsList();
+                            Navigator.of(context).pop();
+                          }
+                        },
+                        icon: const Icon(
+                          Icons.arrow_back_ios,
+                          color: EaselAppTheme.kGrey,
+                        ),
+                      )),
+                )),
+            ValueListenableBuilder(
+              valueListenable: homeViewModel.currentPage,
+              builder: (_, int currentPage, __) {
+                return Text(
+                  homeViewModel.pageTitles[homeViewModel.currentPage.value],
+                  style: Theme.of(context).textTheme.bodyText1!.copyWith(fontSize: 18.sp, fontWeight: FontWeight.w400, color: EaselAppTheme.kDarkText),
+                );
+              },
+            ),
+          ],
+        ),
+        ScreenResponsive(
+          mobileScreen: (context) => const VerticalSpace(6),
+          tabletScreen: (context) => const VerticalSpace(30),
+        ),
+        Expanded(
+          child: PageView(
+            controller: homeViewModel.pageController,
+            physics: const NeverScrollableScrollPhysics(),
+            onPageChanged: (int page) {
+              homeViewModel.currentPage.value = page;
+              switch (page) {
+                case 0:
+                  homeViewModel.currentStep.value = 0;
+                  break;
+                case 1:
```

Comment:
what's the intention for this case "1" with no statement?

Reply:
This handles the difference in the number of screens and the differens in the design steps.